templates/messages/messages.json

```diff
@@ -1,6 +0,0 @@
-{
-  "commandDescription": "print a greeting and your org IDs",
-  "nameFlagDescription": "name to print",
-  "forceFlagDescription": "example boolean flag",
```

Comment:
I think I liked these json messages better

Reply:
I am showing only the command needed to use and removed the extra two lines that the command displays.
In current plugins, we are showing only the command and not output in examples.